management/server/account.go

```diff
@@ -214,9 +223,14 @@ func (am *DefaultAccountManager) GetAccountByUserOrAccountId(userId, accountId,
 	return nil, status.Errorf(codes.NotFound, "no valid user or account Id provided")
 }
 
+func isNil(i idp.Manager) bool {
```

Comment:
Good catch. One thing, the function name is not matching the logic. You are returning true and it isn't nil.

Reply:
done.